### ISSUE TITLE
Debug 3.2 slowdown

### ISSUE THREAD

**Issue by jgillis** [casadi/casadi#2044]
```python
from casadi import *

x = MX.sym('x',10,6000)
y = MX.sym('y',6000,300)
z = mtimes(x,y)

f = Function('f',[x,y],[z],{"jit":True})

vx = np.random.random((10,6000))
vy = np.random.random((6000,300))

import time
t0 = time.time()
vz = f(vx,vy)
print time.time()-t0
```

**Comment by jgillis** [comment]
It turns out that the binaries have been in _Debug_ mode since `3.2`.

The cause is this change:
https://github.com/casadi/casadi/commit/095f192b3740e0452cbaa98e95eda1fb8a97feff#diff-af3b638bc2a3e6c650974192a53c7291R54

For a clean build, CMAKE_BUILD_TYPE seems to be defined, but set to ""

**Comment by jgillis** [comment]
Fixed in 3.2.2
